### PR TITLE
net-misc/xmrig-mo: Algorithm switching xmrig fork

### PR DESCRIPTION
**Pull request by telans**
For usage with the MoneroOcean mining pool
https://github.com/MoneroOcean/xmrig

### CODE REVIEW

File: net-misc/xmrig-mo/xmrig-mo-6.3.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+KEYWORDS="~amd64"
+IUSE="donate libressl +ssl"
+REQUIRED_USE="libressl? ( ssl )"
+
+DEPEND="
+	dev-libs/libuv:=
+	sys-apps/hwloc:=
+	ssl? (
+		!libressl? ( dev-libs/openssl:= )
+		libressl? ( dev-libs/libressl:= ) )"
```
Comment (00-matt) [inline net-misc/xmrig-mo/xmrig-mo-6.3.3.ebuild:24]:
```suggestion
	ssl? (
		!libressl? ( dev-libs/openssl:= )
		libressl? ( dev-libs/libressl:= )
	)"
```

File: net-misc/xmrig-mo/xmrig-mo-6.3.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+S="${WORKDIR}/xmrig-${PV}-${MO_PV}"
+
+PATCHES=("${FILESDIR}/${PN}-nonotls.patch")
+
+src_prepare() {
+	! use donate && PATCHES+=("${FILESDIR}/${PN}-no-default-donate.patch")
```
Comment (00-matt) [inline net-misc/xmrig-mo/xmrig-mo-6.3.3.ebuild:31]:
```suggestion
	use donate || PATCHES+=("${FILESDIR}/${PN}-no-default-donate.patch")
```

or

```suggestion
	use donate || eapply "${FILESDIR}/${PN}-no-default-donate.patch"
```

**Review by 00-matt — CHANGES_REQUESTED** [review]
Great, thank you! Also please could you add a `Signed-off-by: ...` line to your commit if you can.

**Comment by telans** [comment]
Cheers, updated

**Review by 00-matt — APPROVED** [review]

**Pull request closed without merging**